node-js-basics-101/node_modules/node-static/lib/node-static/util.js

```diff
@@ -0,0 +1,30 @@
+var fs   = require('fs')
+  , path = require('path');
```

Comment:
__[jshint]__
* Comma warnings can be turned off with 'laxcomma'.

<!-- Comment by [SideCI](https://sideci.com) -->